Commands/confidentialledger/_check-name-availability.md

```diff
@@ -0,0 +1,13 @@
+# [Command] _confidentialledger check-name-availability_
+
+To check whether a resource name is available.
+
+## Versions
+
+### [2023-06-28-preview](/Resources/mgmt-plane/L3N1YnNjcmlwdGlvbnMve30vcHJvdmlkZXJzL21pY3Jvc29mdC5jb25maWRlbnRpYWxsZWRnZXIvY2hlY2tuYW1lYXZhaWxhYmlsaXR5/2023-06-28-preview.xml) **Stable**
```

Comment:
Do we need to add the current GA version as well for backward compatibility? Feel free to ignore if not relevant